src/app/Console/Commands/AddCustomRouteContent.php

```diff
@@ -68,22 +67,7 @@ public function handle()
                 $this->error('Could not write to file: '.$path);
             }
         } else {
-            $command = 'php artisan vendor:publish --provider="Backpack\Base\BaseServiceProvider" --tag=custom_routes';
```

Comment:
The fact that Backpack\Base is mentioned here tells me NOBODY has gotten to this bit in a long LONG time 😅